tests/subsys/fs/nvs/src/main.c

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <ztest.h>
+
+#include "flash.h"
+#include "flash_map.h"
+#include "stats.h"
+#include "nvs/nvs.h"
+
+#if USE_PARTITION_MANAGER
+#include <pm_config.h>
+#define TEST_FLASH_AREA_STORAGE_OFFSET PM_MCUBOOT_STORAGE_ADDRESS
+#else
+#define TEST_FLASH_AREA_STORAGE_OFFSET DT_FLASH_AREA_STORAGE_OFFSET
```

Comment:
keep only this line for
```
#define TEST_FLASH_AREA_STORAGE_OFFSET DT_FLASH_AREA_STORAGE_OFFSET
```

rest  are unwanted here.

Reply:
fixed